README.md

```diff
@@ -25,3 +26,22 @@ while  not board.is_game_over() and  not board.can_claim_draw():
 
 engine.quit()
 ```
+
+## Training
+
+To run the training process, first one should install the python project on `venv`:
+```bash
+pip install poetry

```

Comment:
Is it necessary? Or it was necessary only to create the 'pyproject.toml' file?

Reply:
New versions of `pip` can install poetry-based projects without the need for `pip install poetry`